src/filmDetailPage.js

```diff
@@ -0,0 +1,8 @@
+import Vue from "vue";
+import FilmDetailPage from "./FilmDetailPage.vue";
+
+Vue.config.productionTip = false;
+
+new Vue({
```

Comment:
Yes, I agree, this is a temporary solution :) But I decided to do so in order to move through the program step-by-step and not jump over some topics. In the future, it will be replaced by a router.

Reply:
Got it.